src/printer.rs

```diff
@@ -150,6 +164,34 @@ impl Printer {
                 write!(self.out, "{}", c).unwrap();
                 self.out.reset().unwrap();
             }
+            FormatSetting::Json => {
+                if let Outcome::Measured(Measurement { avg, variance }) = outcome {
+                    writeln!(
+                        self.out,
+                        r#"{{ "type": "bench", "name": "{}", "median": {}, "deviation": {} }}"#,
+                        info.name, avg, variance
+                    )
+                    .unwrap();
+                } else {
+                    writeln!(
+                        self.out,
+                        r#"{{ "type": "test", "name": "{}", "event": "{}"{} }}"#,
+                        info.name,
+                        match outcome {
+                            Outcome::Passed => "ok",
+                            Outcome::Failed(_) => "failed",
+                            Outcome::Ignored => "ignored",
+                            Outcome::Measured(_) => unreachable!(),
+                        },
+                        match outcome {
+                            Outcome::Failed(Failed { msg: Some(msg) }) =>
+                                format!(r#", "stdout": "Error: \"{}\"\n""#, msg.escape_default()),
+                            _ => "".into(),
+                        }
```

Comment:
Missing trailing comma. This is true for many places actually. Would be great if you could add the trailing comma everywhere.